src/context.js

```diff
@@ -107,6 +161,31 @@ function EvolvContext(store) {
     return true;
   };
 
+  /**
+   * Persist context key to sessionStorage.
+   *
+   * @param {String} key The key to persist.
+   * @param {boolean} [sessionOnly] If true, the value will only be persisted to sessionStorage.
+   *   Default: true
```

Comment:
The way that I read it, `sessionOnly` defaults to `false`, not `true. 

Also JSDoc supports annotating the default value.
```suggestion
   * @param {boolean} [sessionOnly = false] If true, the value will only be persisted to sessionStorage.
```